app/scripts/controllers/newfromtemplate.js

```diff
@@ -355,6 +354,13 @@ angular.module('openshiftConsole')
             $scope.parameterDisplayNames[parameter.name] = parameter.displayName || parameter.name;
           });
 
+          _.each($scope.template.parameters, function(parameter) {
+            var found = _.find(JSON.parse($routeParams.templateParamsMap), function(param){
+              return param.name === parameter.name;
+            });
```

Comment:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/v1.4/quickstart-templates/nodejs-mongodb.json#L364  If we do that, then users can only override name and value key values.  If we leave it as is, they can override any key values they choose.  Do we want to restrict to only name and value values?